proto/buildbuddy_service.proto

```diff
@@ -55,6 +56,8 @@ service BuildBuddyService {
   // Execution API
   rpc GetExecution(execution_stats.GetExecutionRequest)
       returns (execution_stats.GetExecutionResponse);
+  rpc GetExecutionNode(scheduler.GetExecutionNodeRequest) 
```

Comment:
nit: rename the RPC to GetExecutionNodes, to make it more self-documenting that it returns a list of nodes instead of just one (also rename the Request and Response protos)